include/event_dbus_monitor.hpp

```diff
@@ -0,0 +1,640 @@
+                          dumpId;
+        }
+        else if (dumpType == "system")
+        {
+            eventOrigin =
+                "/redfish/v1/Systems/system/LogServices/Dump/Entries/System_" +
+                dumpId;
+        }
+        else if (dumpType == "resource")
+        {
+            eventOrigin = "/redfish/v1/Systems/system/LogServices/Dump/Entries/"
+                          "Resource_" +
+                          dumpId;
+        }
+        else if (dumpType == "hostboot")
+        {
+            eventOrigin = "/redfish/v1/Systems/system/LogServices/Dump/Entries/"
+                          "Hostboot_" +
+                          dumpId;
+        }
+        else if (dumpType == "hardware")
+        {
+            eventOrigin = "/redfish/v1/Systems/system/LogServices/Dump/Entries/"
+                          "Hardware_" +
+                          dumpId;
+        }
+        else if (dumpType == "sbe")
+        {
+            eventOrigin = "/redfish/v1/Systems/system/LogServices/Dump/Entries/"
+                          "SBE_" +
+                          dumpId;
+        }
+        else
+        {
+            BMCWEB_LOG_ERROR("Invalid dump type received when listening for "
+                             "dump created signal");
+            return;
+        }
+        redfish::EventServiceManager::getInstance().sendEvent(
+            redfish::messages::resourceCreated(), eventOrigin, "LogEntry");
+    }
+}
+
+inline void dumpDeletedSignal(sdbusplus::message::message& msg)
+{
+    BMCWEB_LOG_DEBUG("Dump Deleted - match fired");
+
+    if (msg.is_method_error())
+    {
+        BMCWEB_LOG_ERROR("Dump Deleted signal error");
+        return;
+    }
+
+    std::vector<std::string> interfacesList;
+    sdbusplus::message::object_path objPath;
+
+    msg.read(objPath, interfacesList);
+
+    std::string dumpType;
+    std::string dumpId;
+
+    dbus::utility::getNthStringFromPath(objPath, 3, dumpType);
+    dbus::utility::getNthStringFromPath(objPath, 5, dumpId);
+
+    std::string eventOrigin;
+
+    if (dumpType == "bmc")
+    {
+        eventOrigin = "/redfish/v1/Managers/bmc/LogServices/Dump/Entries/" +
+                      dumpId;
+    }
+    else if (dumpType == "system")
+    {
+        eventOrigin =
+            "/redfish/v1/Systems/system/LogServices/Dump/Entries/System_" +
+            dumpId;
+    }
+    else if (dumpType == "resource")
+    {
+        eventOrigin =
+            "/redfish/v1/Systems/system/LogServices/Dump/Entries/Resource_" +
+            dumpId;
+    }
+    else if (dumpType == "hostboot")
+    {
+        eventOrigin =
+            "/redfish/v1/Systems/system/LogServices/Dump/Entries/Hostboot_" +
+            dumpId;
+    }
+    else if (dumpType == "hardware")
+    {
+        eventOrigin =
+            "/redfish/v1/Systems/system/LogServices/Dump/Entries/Hardware_" +
+            dumpId;
+    }
+    else if (dumpType == "sbe")
+    {
+        eventOrigin =
```

Comment:
Use `boost::urls::format()` for the above eventOrigin

Reply:
std::format() suits better in this case